### PR TITLE
CodeClimate Cleanup

### PR DESCRIPTION
**Pull request by edgibbs**

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by eckdanny-osi** [comment]
🎉 Jenkins GH status checks 👍

File: .codeclimate.yml [hunk 1]
```diff
@@ -6,30 +6,21 @@ checks:
...
     enabled: true
   method-count:
     enabled: true
   method-lines:
     enabled: true
     config:
-      languages:
-        javascript:
-          threshold: 50
+      threshold: 200
```
Comment (eckdanny-osi) [inline .codeclimate.yml]: The default threshold value for `method-lines` is 25. **Looks like our override of 50 was not getting used**.

200 seems a bit high for me, -even for `render()` methods. _We're trying to build narrowly focused components, not page templates_.

Of the (8) `method-line` threshold violations being reported in [codeclimate](https://codeclimate.com/github/ca-cwds/case-management/issues?category=complexity&engine_name%5B%5D=structure&engine_name%5B%5D=duplication):

* (6) of them are in `/Clients` app (which I haven't spent much time in to have formed opinions on understandability vs `method-line`s), and
* (7) of which are `render()` methods

Setting the threshold value 50 would eliminate all but (3) violations (73, 92, and 234). This feels right to me upon looking at source code and trying to understand what the component does at a glance.

Comment (edgibbs) [reply .codeclimate.yml]: OK, I assumed that some render methods just are giant chunks of html sometimes, but it makes sense to me to break them up.  I'll switch it.

**Pull request merged**